hacks/genai-monitoring/README.md

```diff
@@ -204,16 +204,18 @@ Follow these steps:
   
 - Apply the necessary code fix and restart the application.
   
-  - Stop the application.
+  - Stop the application. Press Ctrl+C to stop the running containers in the terminal where they are running.
```

Comment:
Another option here is to have people start this up in the background so that it doesn't take over the terminal. Pros and cons to that because you can't see all the logs, or they will interrupt what you are typing but it's an option.